### ISSUE TITLE
Bug in check for existence of Group field

### ISSUE THREAD

**Issue by mjmckp** [dotnet/machinelearning#1372]
https://github.com/dotnet/machinelearning/blob/263a67b94a0747463d3002d0007dbac5257dd58c/src/Microsoft.ML.LightGBM/LightGbmTrainerBase.cs#L428

Should be
```
ch.Check(factory.Data.Schema.Group != null, "The data for ranking task should have group field.");
```

**Comment by Zruty0** [comment]
Thanks @mjmckp ! Looks like a simple bug to fix, I'll mark it as up-for-grabs

**Comment by malik97160** [comment]
I can work on it.

**Comment by Ivanidzo4ka** [comment]
fixed in #1377
